### DISCUSSION TITLE
October 2024 Survey Results - Question 7

### DISCUSSION THREAD

**Discussion by boc-the-git** [mealie-recipes/mealie#4645] [General]
Placeholder discussion for people to share their thoughts and insights about the survey results.

**Comment by ChipSkylark37** [comment]
I've reviewed some of the previous closed issues and there isn't that much of a discussion for a native iOS app. 

Has this been discussed before and the benefits of a native app vs the PWA?

I know currently for iOS, the biggest drawback for the PWA is that 'keep screen awake' does not work due to a safari bug / limitation that has been known for some time but Apple has not fixed / integrated it into iOS Safari.

The workaround that I know is NOT using the PWA and just a standard Safari tab, which then allows for 'keep screen awake' to function properly.
